test/performance/specs/site-editor.spec.js

```diff
@@ -187,6 +189,58 @@ test.describe( 'Site Editor Performance', () => {
 			}
 		} );
 	} );
+
+	test.describe( 'Navigating', () => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentythree' );
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		const iterations = 5;
+		for ( let i = 1; i <= iterations; i++ ) {
+			test( `Run the test (${ i } of ${ iterations })`, async ( {
+				admin,
+				page,
+				metrics,
+			} ) => {
+				await admin.visitSiteEditor( {
+					path: '/wp_template',
+				} );
+
+				// Wait for the loader overlay to disappear. This is necessary
+				// because the overlay is still visible for a while after the editor
+				// canvas is ready, and we don't want it to affect the typing
+				// timings.
+				await page
+					.locator(
+						// Spinner was used instead of the progress bar in an earlier version of the site editor.
+						'.edit-site-canvas-loader, .edit-site-canvas-spinner'
```

Comment:
This should probably be [moved](https://github.com/WordPress/gutenberg/blob/99c2b2dbf158bf154f363e27725b4a8e2186fb88/packages/e2e-test-utils-playwright/src/admin/visit-site-editor.ts#L71) to the `visitSiteEditor` util. It's consistent with other specs here, though, so also marking it for a follow-up PR.

Reply:
I don't think so because not all site editor pages have a canvas/editor

Reply:
Maybe it could be its own util.

Reply:
Ah yes, I've been working on removing that dependency in [this PR](https://github.com/WordPress/gutenberg/pull/55099/files#diff-5c7b7c2d979885b0b5c1c5c056b81f665bcd8c8374053ee63e43a3731f5db388). When that's gone, we should be good with using only the [`getCanvas` util](https://github.com/WordPress/gutenberg/blob/ade7e648250795e0ea4b89a6a3b905b5c61bce97/test/performance/fixtures/perf-utils.ts#L35-L49) when necessary.